src/Heading.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import hotelImg from "./images/hotel.png";
+
+const Heading = (props) => {
+  return (
+    <React.Fragment>
```

Comment:
Normally, its easier to just be able to use <> instead of React.Fragment in these situations.